src/AppBundle/DataFixtures/ORM/PostFixtures.php

```diff
@@ -108,7 +107,7 @@ private function getRandomTags($numTags = 0)
         return $tags;
     }
 
-    private function getPostContent()
+    public static function getPostContent()
```

Comment:
👎 I'd rather suggest to use PHP Traits instead of making `private` methods `public static` or do simple inheritance. We should always avoid public methods which we have to care about later.